script.js

```diff
@@ -1,190 +1,139 @@
-var app;
-
-// Declaring our constants
-var START_YEAR = 1950;
-var END_YEAR = 2015;
-var MAX_RADIUS = 50;
-var TRANSITION_DURATION = 750;
-
-
-
-// // d3.queue() enables us to load multiple data files. Following the example below, we make
-// // additional .defer() calls with additional data files, and they are returned as results[1],
-//results[2], etc., once they have all finished downloading.
-d3.queue()
-.defer(d3.json, 'data/data.json')
-//.defer(d3.json, 'data/table.csv')
-.awaitAll(function (error, results) {
-if (error) { throw error; }
-app.initialize(results[0]);
-});
-
-app = {
-  data: [],
-  components: [],
-
-  options: {
-    year: START_YEAR
-  },
-
-  initialize: function (data) {
-    app.data = data;
-
-    // Here we create each of the components on our page, storing them in an array
-    app.components = [
-      new Chart('#chart')
-    ];
-
-    // Add event listeners and the like here
-
-    // app.resize() will be called anytime the page size is changed
-    d3.select('window').on('resize', app.resize);
-
-    // For demo purposes, let's tick the year every 750ms
-    function incrementYear() {
-      app.options.year += 1;
-      if (app.options.year > END_YEAR) {
-        app.options.year = START_YEAR;
-      }
-
-      app.update();
-    }
-
-    
-    d3.interval(incrementYear, TRANSITION_DURATION);
-  },
-
-  resize: function () {
-    app.components.forEach(function (c) { if (c.resize) { c.resize(); }});
-  },
-
-  update: function () {
-    app.components.forEach(function (c) { if (c.update) { c.update(); }});
-  }
-}
-
-function Chart(selector) {
-  var chart = this;
-
-  // SVG and MARGINS
-
-  var margin = {
-    top: 15, right: 15, bottom: 40, left: 45
+var screenWidth = $(window).width(),
+  mobileScreen = (screenWidth > 400 ? false : true);
```

Comment:
Note that this will only ever be check once, when the page first loads. In order to be truly responsive, we'll need to listen for the `reload` event and update all of the things dependent on these widths, as shown in last week's demo.